x-pack/plugins/stack_alerts/server/rule_types/index_threshold/rule_type.ts

```diff
@@ -49,7 +48,7 @@ export function getRuleType(
   const actionVariableContextDateLabel = i18n.translate(
     'xpack.stackAlerts.indexThreshold.actionVariableContextDateLabel',
     {
-      defaultMessage: 'The date the alert exceeded the threshold.',
+      defaultMessage: 'The date the threshold was exceeded.',
```

Comment:
```suggestion
      defaultMessage: 'The date the alert met the threshold conditions.',
```

The Index threshold rule type allows specifying < threshold condition, so exceeded is not strictly accurate in all cases.